pushy/src/main/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -605,6 +628,26 @@ public ApnsPayloadBuilder setSummaryArgumentCount(final Integer summaryArgumentC
         return this;
     }
 
+    /**
+     * <p>Sets the interruption level.</p>
+     *
+     * <p>Accepted values are {@link INTERRUPTION_LEVEL_PASSIVE}, {@link INTERRUPTION_LEVEL_ACTIVE}, {@link INTERRUPTION_LEVEL_TIME_SENSITIVE} and {@link INTERRUPTION_LEVEL_CRITICAL}.</p>
+     *
+     * @param interruptionLevel the interruption level.
+     *
+     * @return a reference to this payload builder.
+     *
+     * @see @see <a href="https://developer.apple.com/design/human-interface-guidelines/ios/system-capabilities/notifications/">Notifications</a>
+     *
+     * @since 0.14.3
```

Comment:
I think the next release is actually going to be 0.15; can we change this to match?